bedrock/privacy/redirects.py

```diff
@@ -20,4 +20,8 @@
 
     # mozilla/bedrock/#5745 - archive cliqz policy
     redirect(r'^privacy/firefox-cliqz/?$', 'privacy.archive.firefox-cliqz-2018-06'),
+
+    # mozilla/bedrock/#7983
+    redirect(r'^/privacy/products/', 'firefox.privacy.products'),
```

Comment:
This redirect fails when the trailing slash is omitted.
```suggestion
    redirect(r'^/privacy/products/?$', 'firefox.privacy.products'),
```